docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -238,6 +250,17 @@ Network object specifies service IPAM as *external*:
       service:
         ipam: external
 
+#### User hints
+
+User can provide hints about service IP address.
+
+For Arktos managed pool, it will look at the specific annotation of service object like below
+```arktos.futurewei.com/vip-hint: {"cidr":"10.10.1.0/26"}```
```

Comment:
Shall we include this vip-hint when creating service instead? Mizar will assign an ip address to a service, and the ip address needs to fall within the range of vip-hint. correct?

Reply:
sorry this had caused some confusion. 
I've remove the vip-hint at this stage; so mizar can assign vip decided by itself.